src/Symfony/Component/VarDumper/Dumper/AbstractDumper.php

```diff
@@ -90,7 +90,7 @@ public function setCharset(string $charset): string
         $prev = $this->charset;
 
         $charset = strtoupper($charset);
-        $charset = null === $charset || 'UTF-8' === $charset || 'UTF8' === $charset ? 'CP1252' : $charset;
+        $charset = 'UTF-8' === $charset || 'UTF8' === $charset ? 'CP1252' : $charset;
```

Comment:
As an idea, there is no check for empty charset. Though I didn't check how and when this method is used.

```suggestion
        $charset = '' === $charset || 'UTF-8' === $charset || 'UTF8' === $charset ? 'CP1252' : $charset;
```

Reply:
I prefer to only make the strictly minimum changes. In this case, we have `$charset = strtoupper($charset);` and we know hat `strtoupper()` only returns `string`, so we can safely remove the `null === $charset` check. Other than that, I'd prefer to not change anything..